### ISSUE TITLE
Orbital Flight With at Least 3 Crew Not Awarding Confidence

### ISSUE THREAD

**Issue by CXG-2827** [KSP-RO/RP-1#2221]
Reputation is awarded but the 750 Confidence does not show in either the Tally nor on the Completed contract message. Confidence remains at 219 and does not increment from completion.

Attempted to force complete contract in F12 menu after reloading earlier save and still did not receive confidence.

Included quicksave prior to de-orbiting craft for contract completion. Ran with Express install and RN Soviet Spacecraft
![screenshot4](https://github.com/KSP-RO/RP-1/assets/5134828/898f730c-9e18-4df7-a28f-a687bcbb4e33)
![screenshot5](https://github.com/KSP-RO/RP-1/assets/5134828/f52289ed-380b-464b-9150-018c4541a770)
[quicksave.zip](https://github.com/KSP-RO/RP-1/files/12569501/quicksave.zip)

**Comment by NathanKell** [comment]
You don't have either of the programs for which this is a contract. How was it even available for you? Did you accept it prior to completing Advanced Crewed but then completed the program, and the contract didn't go away?

**Comment by CXG-2827** [comment]
I just remembered. I had advanced crewed orbit active when I accepted the context, and launched the flight.

After completing first docking, I returned to the space center and noticed I was able to closeout the program, so I ✔️ adv crewed orbit.

In this case, shouldn't it force cancel the contract?

**Comment by NathanKell** [comment]
Yes, it should, and it used to. Not sure why it didn't. Moving to CC.
